ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/tables/ethernet-interfaces.md

```diff
@@ -1156,6 +1184,77 @@
                 #   - 1-10
               - range: <str; required; unique>
                 mac_address_maximum: <int>
+          trunk:
+
+            # VLAN ID or range of VLAN IDs.
+            # Warning: This should not be combined with `ethernet_interface.mode = trunk`.
+            allowed_vlans: <str>
+
+            # Set native VLAN when interface is in trunking mode.
+            # Warning: This should not be combined with `ethernet_interfacse[].native_vlan`.
+            native_vlan: <int; 1-4094>
+
+            # If setting both native_vlan and native_vlan_tag, native_vlan_tag takes precedence.
+            # Warning: This should not be combined with `ethernet_interfacse[].native_vlan_tag`.
+            native_vlan_tag: <bool>
+
+            # Enable secondary VLAN mapping for a private vlan.
+            # Warning: This should not be combined with `ethernet_ineterfaces[].trunk_private_vlan_secondary`.
+            private_vlan_secondary: <bool>
+            groups:
+
+                # Trunk group name.
+                # Warning: This should not be combined with `ethernet_ineterfaces[].trunk_groups`.
+              - <str>
+
+          # Set VLAN when interface is in access mode.
+          # Warning: This should not be combined withe `ethernet_interfaces[].mode = access/dot1q-tunnel`.
+          access_vlan: <int>
+          phone:
+
+            # Warning: This should not be combined with `ethernet_interfaces[].phone.vlan`.
+            vlan: <int; 1-4094>
+
+            # Warning: This should not be combined with `ethernet_interfaces[].phone.trunk`
+            trunk: <str; "tagged" | "tagged phone" | "untagged" | "untagged phone">
+
+          # Warning: This should not be combined with `ethernet_interfaces[].mode`
+          mode: <str; "access" | "dot1q-tunnel" | "trunk" | "trunk phone">
```

Comment:
```
wan1-dc1(config-if-Et1)#switchport mode ?
  access        Set trunking mode to ACCESS unconditionally
  dot1q-tunnel  Set trunking mode to DOT1Q-TUNNEL unconditionally
  tap           Set switchport mode to TAP unconditionally
  tap-tool      Set switchport mode to both Tap and Tool
  tool          Set switchport mode to TOOL unconditionally
  trunk         Set trunking mode to TRUNK unconditionally
```

Reply:
I think we should focus on our current functionality first, but off course not lock out future enhancements.

Reply:
Sure!